dashboard/test/models/contact_rollups_processed_test.rb

```diff
@@ -185,6 +187,28 @@ class ContactRollupsProcessedTest < ActiveSupport::TestCase
     end
   end
 
+  test 'extract_hoc_organizer_years' do
+    contact_data = {
+      'pegasus.forms' => {
+        'kind' => [
+          # most common value type 'HocSignup<year>'
+          {'value' => 'HocSignup2019'},
+          # duplicate values
+          {'value' => 'CSEdWeekEvent2013'},
+          {'value' => 'CSEdWeekEvent2013'},
+          # invalid values
+          {'value' => nil},
+          {'value' => 'HocSignup'},
+          {'value' => 'HocCensus2017'}
+        ]
+      }
+    }
+    expected_output = {hoc_organizer_years: '2013,2019'}
```

Comment:
We don't have any multiselect fields where the field would have a comma in them, do we?

Reply:
We don't. There are only 4 multi-select fields (pd enrolled, pd attendend, HoC years, and roles) and we control the values put in them.